hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/config/ConfigServer.java

```diff
@@ -150,5 +151,13 @@ private String getJournalNodes(List<String> journalNodes) {
       }
       return journalNodeString;
     }
+
+    private String nameNodeId(String nnNum) {
```

Comment:
Instead of adding this logic let's just pass namenode1 or namenode2 as the parameter, i.e. nn=namenode1